compiler/modules/front.syntax/token.bal

```diff
@@ -487,6 +490,14 @@ public readonly class SourceFile {
         return unpackPosition(pos);
     }
 
+    public function lineContent(Position pos) returns string {
```

Comment:
No! You're supposed to reconstruct it from the ScannedLine.

Reply:
fixed